stablehlo/reference/Element.cpp

```diff
@@ -120,6 +120,43 @@ Element Element::operator+(const Element &other) const {
   report_fatal_error(std::move(err));
 }
 
+Element Element::operator-(const Element &other) const {
+  auto type = getType();
+  assert(type == other.getType());
+
+  if (isSupportedFloatType(type)) {
+    auto left = getFloatValue(*this);
+    auto right = getFloatValue(other);
+    return Element(type, FloatAttr::get(type, left - right));
+  }
+
+  if (isSupportedIntegerType(type)) {
+    auto left = getIntegerValue(*this);
+    auto right = getIntegerValue(other);
+    return Element(type, IntegerAttr::get(type, left - right));
+  }
+
+  if (isSupportedComplexType(type)) {
+    auto complexElemTy = type.cast<ComplexType>().getElementType();
+    auto leftComplexValue = getComplexValue(*this);
+    auto rightComplexValue = getComplexValue(other);
+
+    return Element(
+        type,
+        ArrayAttr::get(
+            type.getContext(),
+            {FloatAttr::get(complexElemTy,
+                            leftComplexValue.real() - rightComplexValue.real()),
+             FloatAttr::get(complexElemTy, leftComplexValue.imag() -
+                                               rightComplexValue.imag())}));
+  }
+
+  // Report error.
```

Comment:
The comment pretty much repeats what the code below does (`report_fatal_error`), so we should drop it.

Reply:
Resolving comment since it's fixed in #190